release_notes/index.adoc

```diff
@@ -20,3 +20,11 @@ technical notes, see the
 https://docs.openshift.com/enterprise/3.1/release_notes/index.html[OpenShift
 Enterprise 3.1 Release Notes].
 endif::[]
+
+[[release-versioning-policy]]
+== Versioning Policy
+{product-title} does not have non-backwards compatible changes and, therefore,
```

Comment:
double negative, suggest  `product provides strict backwards compatibility guarantees for all supported APIs (excluding alpha APIs, which may be changed without notice, and beta APIs, which may occasionally be changed in a non-backwards compatible manner) and will continue to support older clients against newer servers.  Changes of APIs for non-security related reasons will involve at minimum two minor releases (3.1 -> 3.2 -> 3.3, for example) to allow older clients to update.  Utilizing new capabilities may require newer clients.